pages.ml/linux/nmcli.md

```diff
@@ -1,7 +1,7 @@
 # nmcli
 
 > നെറ്റ്‌വർക്ക് മാനേജർ നിയന്ത്രിക്കാൻ വേണ്ടിയുള്ള കമാൻഡ് ലൈൻ ഉപകരണം.
-> കൂടുതൽ വിവരങ്ങൾ: <https://man.archlinux.org/man/nmcli.1>.
+> കൂടുതൽ വിവരങ്ങൾ: <https://manned.org/nmcli>.
```

Comment:
```suggestion
> കൂടുതൽ വിവരങ്ങൾ: <https://developer.gnome.org/NetworkManager/stable/nmcli>.
```
According to Wikipedia, <https://wiki.gnome.org/Projects/NetworkManager> is the official page for `NetworkManager`, which then points to <https://developer.gnome.org/NetworkManager/stable/nmcli> for `nmcli`.

Reply:
Nice! Thanks, will update.